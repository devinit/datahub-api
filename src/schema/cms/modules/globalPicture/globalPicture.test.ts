import {getGlobalPictureThemes, getTheme} from '.';
import * as prettyFormat from 'pretty-format';

describe('Site global Picture narratives and refrence module', () => {
    it('should return all themes', async () => {
        const themes = await getGlobalPictureThemes();
        const theme = getTheme('poverty', themes);
        expect(themes[0].id).toBeDefined();
        expect(themes.length).toBeGreaterThan(4);
        expect(prettyFormat(theme)).toMatchSnapshot();
    }, 20000);
});
