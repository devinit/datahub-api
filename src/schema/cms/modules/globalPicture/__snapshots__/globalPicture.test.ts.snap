// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Site global Picture narratives and refrence module should return all themes 1`] = `
"Object {
  \\"default\\": \\"poor-people-190\\",
  \\"id\\": \\"poverty\\",
  \\"name\\": \\"Poverty\\",
  \\"position\\": \\"1\\",
}"
`;
